presto-resource-group-managers/src/main/java/com/facebook/presto/resourceGroups/db/DbResourceGroupConfigurationManager.java

```diff
@@ -137,25 +139,30 @@ public void configure(ResourceGroup group, SelectionContext context)
     @VisibleForTesting
     public synchronized void load()
     {
-        Map.Entry<ManagerSpec, Map<ResourceGroupIdTemplate, ResourceGroupSpec>> specsFromDb = buildSpecsFromDb();
-        ManagerSpec managerSpec = specsFromDb.getKey();
-        Map<ResourceGroupIdTemplate, ResourceGroupSpec> resourceGroupSpecs = specsFromDb.getValue();
-        Set<ResourceGroupIdTemplate> changedSpecs = new HashSet<>();
-        Set<ResourceGroupIdTemplate> deletedSpecs = Sets.difference(this.resourceGroupSpecs.keySet(), resourceGroupSpecs.keySet());
+        try {
+            Map.Entry<ManagerSpec, Map<ResourceGroupIdTemplate, ResourceGroupSpec>> specsFromDb = buildSpecsFromDb();
+            ManagerSpec managerSpec = specsFromDb.getKey();
+            Map<ResourceGroupIdTemplate, ResourceGroupSpec> resourceGroupSpecs = specsFromDb.getValue();
+            Set<ResourceGroupIdTemplate> changedSpecs = new HashSet<>();
+            Set<ResourceGroupIdTemplate> deletedSpecs = Sets.difference(this.resourceGroupSpecs.keySet(), resourceGroupSpecs.keySet());
 
-        for (Map.Entry<ResourceGroupIdTemplate, ResourceGroupSpec> entry : resourceGroupSpecs.entrySet()) {
-            if (!entry.getValue().sameConfig(this.resourceGroupSpecs.get(entry.getKey()))) {
-                changedSpecs.add(entry.getKey());
+            for (Map.Entry<ResourceGroupIdTemplate, ResourceGroupSpec> entry : resourceGroupSpecs.entrySet()) {
+                if (!entry.getValue().sameConfig(this.resourceGroupSpecs.get(entry.getKey()))) {
+                    changedSpecs.add(entry.getKey());
+                }
             }
-        }
 
-        this.resourceGroupSpecs = resourceGroupSpecs;
-        this.cpuQuotaPeriod.set(managerSpec.getCpuQuotaPeriod());
-        this.rootGroups.set(managerSpec.getRootGroups());
-        this.selectors.set(buildSelectors(managerSpec));
+            this.resourceGroupSpecs = resourceGroupSpecs;
+            this.cpuQuotaPeriod.set(managerSpec.getCpuQuotaPeriod());
+            this.rootGroups.set(managerSpec.getRootGroups());
+            this.selectors.set(buildSelectors(managerSpec));
 
-        configureChangedGroups(changedSpecs);
-        disableDeletedGroups(deletedSpecs);
+            configureChangedGroups(changedSpecs);
+            disableDeletedGroups(deletedSpecs);
+        }
+        catch (Exception e) {
+            log.error("Error loading configuration from db: %s", e);
```

Comment:
You normally want to pass the exception as the exception parameter so it gets logged with the full stack trace. This will only print the name and message.